.husky/pre-commit

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn pretty-quick --staged
+yarn lint-staged
```

Comment:
The `lint-staged` program throws an error after I made a simple docs change. I don't think I'm breaking anything here.

**Steps to reproduce:**

1. Open new codespace on this PR branch
1. Open file `docs/usage/configuration-templates.md
1. Put new text on line 9 of the file: `Test test test.`
1. Stage the change with `git add .`
1. Try to create a commit with `git commit -m "Test"`
1. `lint-staged` runs tasks for staged files and throws an error

**Error message:**

```bash
root@codespaces-7b782b:/workspaces/renovate# git commit -m "Test"
yarn run v1.22.19
$ /workspaces/renovate/node_modules/.bin/lint-staged
✔ Preparing lint-staged...
❯ Running tasks for staged files...
  ❯ .lintstagedrc — 1 file
    ❯ * — 1 file
      ✖ ls-lint [KILLED]
    ❯ *.md — 1 file
      ✖ markdown-lint [ENOENT]
    ↓ **/*.{ts,js,mjs,cjs} — no files [SKIPPED]
    ❯ **/*.{ts,js,mjs,json,md,yml} — 1 file
      ✖ prettier [KILLED]
↓ Skipped because of errors from tasks. [SKIPPED]
✔ Reverting to original state because of errors...
✔ Cleaning up temporary files...

✖ markdown-lint failed without output (ENOENT).

✖ ls-lint failed without output (KILLED).

✖ prettier failed without output (KILLED).
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
husky - pre-commit hook exited with code 1 (error)
root@codespaces-7b782b:/workspaces/renovate# 
```